docs/index.md

```diff
@@ -757,7 +757,59 @@ If you use a virtualenv for your current project, add a path to it in your [proj
 }
 ```
 
-See: [github:palantir/python-language-server](https://github.com/palantir/python-language-server)
+A pyls configuration in `LSP.sublime-settings` that is convenient for novices:
+
+```js
+{
+  //...
+  
+	"clients":
+	{
+        "pyls": {
+            "enabled": true,
+            "command": ["pyls"],
+            "scopes": ["source.python"],
+            "syntaxes": ["Packages/Python/Python.sublime-syntax", "Packages/MagicPython/grammars/MagicPython.tmLanguage", "Packages/Djaneiro/Syntaxes/Python Django.tmLanguage"],
+            "languageId": "python",
+            "settings": {
+                "pyls": {
+                    "configurationSources": ["flake8"],
+                    "plugins": {
+                        "jedi": {
+                            "enabled": true,  // Turn on automatic completion by Jedi
+                            "extra_paths": [
+                                // "The directory where the pip installation package is located",
+                            ],
+                        },
+                        "pyflakes": {"enabled": true,},
+                        "pycodestyle": {
+                            "enabled": true,
+                            "ignore": [
+                                "E501",   // ignore lint
```

Comment:
I'm not sure we should be included opinionated rules in suggested config.
I would at least comment it out. And replace current comment with `Line too long (82 > 79 characters)` as the current one is not helpful.